datasources/src/main/scala/com/normation/plugins/datasources/Serialization.scala

```diff
@@ -100,7 +101,7 @@ object DataSourceJsonSerializer{
                       case MissingNodeBehavior.Delete              =>  ("name"  -> MissingNodeBehavior.Delete.name):JObject
                       case MissingNodeBehavior.NoChange            =>  ("name"  -> MissingNodeBehavior.NoChange.name):JObject
                       case MissingNodeBehavior.DefaultValue(value) => (("name"  -> MissingNodeBehavior.DefaultValue.name) ~
-                                                                       ("value" -> value )):JObject
+                                                                       ("value" -> parse(value.render(ConfigRenderOptions.concise())))):JObject
```

Comment:
you use that one several time (in and in `ConfigValueSerializer`). I think it should be at minima a dedicated method, and even better a full method in `GenericProperty` - especially since it's used in `implicit class RenderProperty#jsonValue` (class name is bad and should be changed). So: make `jsonValue` a dedicated `def toJsonValue(value: ConfigValue): JValue`, use it in ``implicit class RenderProperty`, use it here.

Reply:
In fact there is a serializeToJson that exists in GenericProperty, we should use it instead

And you may be right and we should use it too in Rudder